.github/workflows/ci.yml

```diff
@@ -16,7 +16,7 @@ jobs:
   tox:
     strategy:
       matrix:
-        os: [ubuntu-latest] #[ubuntu-latest, macos-latest, windows-latest]
+        os: [ubuntu-latest, macos-latest] #[ubuntu-latest, macos-latest, windows-latest]
```

Comment:
it's important to have cross-platform tests, but is there a particular reason this is added in this PR? we could open a separate PR adding all OSs and versions (we should probably do this before the workshop in February)

Reply:
yes, makes sense. I'll remove it from this pr, but let's keep it in mind for the future